src/main/java/com/endava/internship/service/UserServiceImpl.java

```diff
@@ -2,68 +2,111 @@
 
 import com.endava.internship.domain.Privilege;
 import com.endava.internship.domain.User;
-import com.endava.internship.service.UserService;
```

Comment:
why it was removed? please describe your opinion under this comment

Reply:
they are in the same package. If I am not mistaken, this is a static import, we do not write it, instead we write the path to the class, and since our class is in the same package, we simply write its name.